### PR TITLE
[PERFSCALE-2464]: Adds DP for ROSA 4.14, 4.15

### PR DESCRIPTION
**Pull request by krishvoor**
This PR allows the PerfScale team to periodically schedule 4.14/4.15 nightly-based Data-Path tests.

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml:42]: think we want tot use _aws-perfscale-qe_ profile here

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml:42]: You are right anything <24 nodes will use `aws-perfscale-qe`

**Comment by paigerube14** [comment]
I'm not sure on the sizing of the rosa nodes, I'll let someone else comment on that. I think after you do `make jobs` this looks good to me.

**Comment by krishvoor** [comment]
> I'm not sure on the sizing of the rosa nodes, I'll let someone else comment on that. I think after you do `make jobs` this looks good to me.

This was picked from existing 4.13 Data path tests:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml#L216-L231

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Comment by svetsa-rh** [comment]
@krishvoor change looks good to me. I tried to view the test result failure, but getting prow timeouts.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Comment by SachinNinganure** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Comment by SachinNinganure** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-perfscale-rosa-multiaz-data-path-9nodes-periodic

**Review by jtaleric — CHANGES_REQUESTED** [review]
Need @rsevilla87 to provide his .02 before merge.

**Comment by SachinNinganure** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-perfscale-rosa-multiaz-data-path-9nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:45]: Updating this to m5.2xlarge

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:45]: Not sure, how to enforce changes to infra nodes since this is ROSA.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:53]: `- ref: openshift-qe-perfscale-aws-data-path-sg`

Need to add this to ensure we can run network tests using hostNetwork.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml:69]: `- ref: openshift-qe-perfscale-aws-data-path-sg`

Need to add this to ensure we can run network tests using hostNetwork.

**Review by jtaleric — CHANGES_REQUESTED** [review]

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml
Comment (jtaleric) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml:69]: You have the ordering wrong. the path-sg needs to be first.

**Review by jtaleric — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml:70]: Ack, updated ptal

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Comment by krishvoor** [comment]
[4.14 DP](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43240/rehearse-43240-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes/1742341426403872768) Test failure is not related to PR in itself, the smoke test passed, where-as the overall failure was from Prow.

**Comment by krishvoor** [comment]
4.15 DP Test failure is because of `Forbidden - CSRF token invalid`

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:73]: `{"component":"entrypoint","file":"k8s.io/test-infra/prow/entrypoint/run.go:169","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 5h0m0s timeout","severity":"error","time":"2024-01-03T11:00:50Z"}`

We might want to bump this timeout to 6h

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml]: Bumped the timeout, ptal

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Comment by krishvoor** [comment]
Blocked on https://github.com/openshift/release/pull/47359

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-data-path-9nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-nightly-x86-data-path-9nodes

**Review by jtaleric — APPROVED** [review]
/lgtm

**Pull request merged**
